modules/vector-sets/vset.c

```diff
@@ -2067,54 +2067,305 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
+
+    // Register command VREM
     if (RedisModule_CreateCommand(ctx,"VREM",
         VREM_RedisCommand,"write",1,1,1) == REDISMODULE_ERR)
         return REDISMODULE_ERR;
 
+    RedisModuleCommand *vrem_cmd = RedisModule_GetCommand(ctx, "VREM");
+    if (vrem_cmd == NULL) return REDISMODULE_ERR;
+
+    RedisModuleCommandArg vrem_args[] = {
+        { .name = "key", .type = REDISMODULE_ARG_TYPE_KEY, .key_spec_index = 0 },
+        { .name = "element", .type = REDISMODULE_ARG_TYPE_STRING, .flags = REDISMODULE_CMD_ARG_MULTIPLE },
+        { NULL }
+    };
+    RedisModuleCommandInfo vrem_info = {
+        .version = REDISMODULE_COMMAND_INFO_VERSION,
+        .summary = "Remove one or more elements from a vector set",
+        .since = "8.0.0",
+        .arity = -2,
+        .args = vrem_args,
+    };
+    if (RedisModule_SetCommandInfo(vrem_cmd, &vrem_info) == REDISMODULE_ERR) return REDISMODULE_ERR;
+
+    // Register command VSIM
     if (RedisModule_CreateCommand(ctx,"VSIM",
         VSIM_RedisCommand,"readonly",1,1,1) == REDISMODULE_ERR)
         return REDISMODULE_ERR;
 
+    RedisModuleCommand *vsim_cmd = RedisModule_GetCommand(ctx, "VSIM");
+    if (vsim_cmd == NULL) return REDISMODULE_ERR;
+
+    RedisModuleCommandArg vsim_args[] = {
+        { .name = "key", .type = REDISMODULE_ARG_TYPE_KEY, .key_spec_index = 0 },
+        { .name = "format", .type = REDISMODULE_ARG_TYPE_ONEOF, .subargs = (RedisModuleCommandArg[]) {
+                { .name = "ele", .type = REDISMODULE_ARG_TYPE_PURE_TOKEN, .token = "ELE" },
+                { .name = "fp32", .type = REDISMODULE_ARG_TYPE_PURE_TOKEN, .token = "FP32" },
+                { .name = "values", .type = REDISMODULE_ARG_TYPE_PURE_TOKEN, .token = "VALUES" },
+                { NULL }
+            }
+        },
+        { .name = "vector_or_element", .type = REDISMODULE_ARG_TYPE_STRING },
+        { .name = "withscores", .type = REDISMODULE_ARG_TYPE_PURE_TOKEN, .token = "WITHSCORES", .flags = REDISMODULE_CMD_ARG_OPTIONAL },
+        { .name = "withattribs", .type = REDISMODULE_ARG_TYPE_PURE_TOKEN, .token = "WITHATTRIBS", .flags = REDISMODULE_CMD_ARG_OPTIONAL },
+        { .name = "count", .type = REDISMODULE_ARG_TYPE_INTEGER, .token = "COUNT", .flags = REDISMODULE_CMD_ARG_OPTIONAL },
+        { .name = "max_distance", .type = REDISMODULE_ARG_TYPE_DOUBLE, .token = "EPSILON", .flags = REDISMODULE_CMD_ARG_OPTIONAL },
+        { .name = "search-exploration-factor", .type = REDISMODULE_ARG_TYPE_INTEGER, .token = "EF", .flags = REDISMODULE_CMD_ARG_OPTIONAL },
+        { .name = "expression", .type = REDISMODULE_ARG_TYPE_STRING, .token = "FILTER", .flags = REDISMODULE_CMD_ARG_OPTIONAL },
+        { .name = "max-filtering-effort", .type = REDISMODULE_ARG_TYPE_INTEGER, .token = "FILTER-EF", .flags = REDISMODULE_CMD_ARG_OPTIONAL },
+        { .name = "truth", .type = REDISMODULE_ARG_TYPE_PURE_TOKEN, .token = "TRUTH", .flags = REDISMODULE_CMD_ARG_OPTIONAL },
+        { NULL }
```

Comment:
Optional `NOTHREAD`  token arg is documented and tested and not listed here.

Reply:
Was just following commands.json. Now added in both places, code and commands.json.